PP_APP/server/frontend/templates/rate.html

```diff
@@ -1,60 +1,92 @@
-                    <div class="text-dropzone text-center align-middle">
-                        <h2>TOP VIEW</h2>
-                        <p>(Click OR Drag and Drop)</p>
-                    </div>
-                    <input type="file" hidden id="top_img" required>
-                </label>
+   <div class="container">
+   <div class="row welcome text-center">
+      <div class="col-12">
+         <h1 class="display-8 mb-4">How to Rate Your Package?</h1>
+      </div>
+      <hr>
+   </div>
+   <div class="row">
+      <div class="col-sm">
+         <div class="card mb-5" style="border-color:transparent">
+            <div class="card-body">
+               <h2 class="card-title">Step 1.</h2>
+               <p class="card-text">Take a picture from the TOP of your package.</p>
             </div>
-            <div class="col-sm">
-                <label id="side-dropzone" class="dropzone side-dropzone" style="background: url({% static 'img/sample_side.png' %}) top left no-repeat;background-size:cover;">
-                    <div class="text-center align-middle">
-                        <div class="text-dropzone text-align-center">
-                            <h2>LOGO VIEW</h2>
-                            <p>(Click OR Drag and Drop)</p>
-                        </div>
-                    </div>
-                    <input type="file" hidden id="side_img" required>
-                </label>
+            <img class="card-img-top" src="{% static 'img/sample_top.png' %}" height="250">
+         </div>
+      </div>
+      <div class="col-sm">
+         <div class="card mb-5" style="border-color:transparent">
+            <div class="card-body">
+               <h2 class="card-title">Step 2.</h2>
+               <p class="card-text">Take a picture of the LOGO of your package.</p>
             </div>
-        </div>
-
-        <div class="form-group">
-            <label for="exampleInputEmail1">Your Email</label>
-            <input type="email" class="form-control" id="user_email" aria-describedby="emailHelp"
-                placeholder="Enter your email address"  required>
-            <small id="emailHelp" class="form-text text-muted">This cannot be empty.</small>
-        </div>
-
-        <div class="row">
-            <div class="col justify-content-center">
-                <input type="submit" id="btnSubmit" name="submit" value="Rate My Package!" class="btn btn-primary btn-lg">
+            <img class="card-img-top" src="{% static 'img/sample_side.png' %}" height="250">
+         </div>
+      </div>
+      <div class="col-sm">
+         <div class="card mb-5" style="border-color:transparent">
+            <div class="card-body">
+               <h2 class="card-title">Step 3.</h2>
+               <p class="card-text">Click submit and you will receive your result and score!</p>
             </div>
-        </div>
-    </form>
-    </div>
-
-    </div>
+            <img class="card-img-top" src="{% static 'img/fig.png' %}" height="250">
```

Comment:
I think the style of all 3 pictures should match. The first two have a grainy background with a large orange dashed border around it and the third doesn't. I think both styles are good, but it should be consistant.

Reply:
the third is the result image, so it should be the way it is otherwise could be confusing to the user.  The orange frames for the first and second images is refer to the upload section below this step section. thats why only the first two images has the fame thing.